Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -769,41 +769,41 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
           "yesorno", _("Activated ?"), "Is the antialiasing activated ?");
   obj.AddCondition(
          "IsAntialiasingOn",
-         _("Antialiasing activated on the shape drawer"),
-         _("Checks if the antialiasing is activated for a shape drawer"),
-         _("Is the antialiasing on for _PARAM0_ ?"),
+         _("Anti-aliasing activated on the shape drawer"),
+         _("Checks if the anti-aliasing is activated for a shape drawer"),
+         _("Is the anti-aliasing enabled on _PARAM0_ ?"),
```

Comment:
We should be consistent on using `activated` or `enabled`.  I like `enabled` but they are very similar so you can choose as long as they match everywhere.